deps/wazuh_testing/wazuh_testing/process_resource_monitoring/src/wazuh_metrics.py

```diff
@@ -0,0 +1,333 @@
+# Copyright (C) 2015-2024, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+"""Script to monitor Wazuh central components resource usage."""
+
+import argparse
+import logging
+from collections import defaultdict
+from datetime import datetime
+from os import makedirs
+from os.path import join
+from signal import SIGINT, SIGTERM, signal
+from tempfile import gettempdir
+from time import sleep, time_ns
+from types import FrameType
+from typing import Dict, List, Optional
+
+from process_resource_monitoring._logger import logger
+from process_resource_monitoring.disk_usage_tracker import DiskUsageTracker
+from process_resource_monitoring.monitor import Monitor
+
```

Comment:
Same as [here](https://github.com/wazuh/wazuh-qa/pull/5747#discussion_r1768913070). 
It seems like if there is a class after the import block two spaces are required, otherwise only one.